src/deepsparse/pipeline.py

```diff
@@ -277,11 +279,13 @@ def __call__(self, *args, monitoring: bool = False, **kwargs) -> BaseModel:
             identifier=InferencePhases.POST_PROCESS,
             value=timer.time_delta(InferencePhases.POST_PROCESS),
             category=MetricCategories.SYSTEM,
+            identifier_pre_fix="prediction_latency",
```

Comment:
IMO the most controversial thing in this PR. Willing to defend this design, but am aware of the ugliness.

Reply:
This enables us to match system logs in the following way:
e.g.
identifier = `prediction_latency/pipeline_name/pre_processing`
vs
target_identifier = `prediction_latency` (from the system logging config)